packages/bot-web-ui/src/stores/transactions-store.js

```diff
@@ -1,13 +1,30 @@
-import { action, observable } from 'mobx';
+import { action, observable, reaction } from 'mobx';
 import { formatDate, isEnded } from '@deriv/shared';
 import { transaction_elements } from '../constants/transactions';
 
 export default class TransactionsStore {
     constructor(root_store) {
         this.root_store = root_store;
+        this.transaction_storage_key = 'transaction_cache';
+        this.get_transaction_elements = JSON.parse(sessionStorage.getItem(this.transaction_storage_key));
```

Comment:
Same here, try to create a function for retrieving the transaction elements from the sessionStorage.